docs/framework/windows-workflow-foundation/consuming-odata-feeds-from-a-workflow.md

```diff
@@ -40,14 +40,17 @@ WCF Data Services is a component of the [!INCLUDE[dnprdnshort](../../../includes
  [!code-csharp[CFX_WCFDataServicesActivityExample#10](../../../samples/snippets/csharp/VS_Snippets_CFX/CFX_WCFDataServicesActivityExample/cs/Program.cs#10)]  
   
  When this workflow is invoked, the following data is written to the console:  
-  
- **Calling WCF Data Service...**  
-**8/25/1997**   
-**10/3/1997**   
-**10/13/1997**   
-**1/15/1998**   
-**3/16/1998**   
-**4/9/1998**    
+
+```
```

Comment:
You could tag this as console.
A better solution is to include the code with the sample, though that involves opening a new PR in samples. That's our usual convention, which has the advantage of combining output and the code that produces it in a single place.